fastestimator/backend/l2_regularization.py

```diff
@@ -38,9 +38,10 @@ def l2_regularization(model: Union[tf.keras.Model, torch.nn.Module], beta: float
         ValueError: If `model` belongs to an unacceptable framework.
     """
     if isinstance(model, torch.nn.Module):
-        l2_loss = torch.tensor(0.)
+        l2_loss = torch.tensor(0.).to(torch.device(next(model.parameters()).device))
```

Comment:
to you need to wrap the device in a new torch.device()? seems like that could be skipped

Reply:
I could use tensor.new(), or 
I could create tensor on the device similiar to other parameters rather than transfering them

Reply:
for example, the following should work:
`a=Tensor1
 b=Tensor2
 b.to(a.device)
`
rather than:
`a=Tensor1
 b=Tensor2
 b.to(torch.device(a.device))
`